tests/unit/components/Editor/ItemEditor.spec.js

```diff
@@ -417,6 +439,18 @@ describe("ItemEditor.vue", () => {
       ).toBeTruthy();
     });
 
+    it("check for survey mode", async () => {
+      await wrapper
+        .find('[data-test="surveyQuestionCheckbox"]')
+        .setChecked("checked");
+      await flushPromises();
+      expect(
+        wrapper.find('[data-test="subjectiveQuestionContainer"]').exists()
+      ).toBeTruthy();
+      expect(wrapper.find('[data-test="options"]').exists()).toBeFalsy();
```

Comment:
this should already not be shown irrespective of survey mode or not right? not needed here?